cli/drivers/Magento2ValetDriver.php

```diff
@@ -96,8 +97,12 @@ public function isStaticFile($sitePath, $siteName, $uri)
         if (strpos($uri, '/static/') !== false) {
             $isMagentoStatic = true;
         }
+        
+        if (strpos($uri, '.xml') !== false) {
+            $isSitemap = true;
```

Comment:
Why a new variable instead of just setting `$isMagentoStatic`? You don't seem to use this variable anywhere other than the static check on line 105 so I'm curious on your reasoning for it?

Reply:
Because of this:
```
        if ($isMagentoStatic) {
            $resource = preg_replace('#static(/version[0-9]+)?/#', '', $uri, 1);
            $uri = '/static' . $resource;
        }
```
The sitemap is not in that location. It should be handled by:
```
        if (file_exists($staticFilePath = $sitePath . '/pub' . $uri)) {
            return $staticFilePath;
        }
```

Reply:
> In fact I wonder why not all documents in pub/ are returned directly if they are existing files. Do you know why that is?
At the time I first created the driver, I was following how some of the other valet drivers worked. None of them handled for static content really well.  I think your latest rewrite of the driver looks pretty good though